exposed-kotlin-datetime/src/main/kotlin/org/jetbrains/exposed/sql/kotlin/datetime/KotlinDateColumnType.kt

```diff
@@ -118,6 +118,18 @@ private fun dateTimeWithFractionFormat(fraction: Int): DateTimeFormatter {
     return DateTimeFormatter.ofPattern(newFormat).withLocale(Locale.ROOT).withZone(ZoneId.systemDefault())
 }
 
+private fun oracleDateTimeLiteral(instant: Instant) =
+    "TO_TIMESTAMP('${SQLITE_AND_ORACLE_DATE_TIME_STRING_FORMATTER.format(instant.toJavaInstant())}', 'YYYY-MM-DD HH24:MI:SS.FF3')"
+
+private fun oracleLocalTimeLiteral(time: LocalTime) =
+    "TO_TIMESTAMP('${ORACLE_TIME_STRING_FORMATTER.format(time.toJavaLocalTime())}', 'YYYY-MM-DD HH24:MI:SS')"
+
```

Comment:
Let's not include any changes related to `timeLiteral()` in this PR.

Reply:
yep, agree, I changed it to

```kotlin
if (dialect is OracleDialect || dialect.h2Mode == H2Dialect.H2CompatibilityMode.Oracle) {
  return "TIMESTAMP '${ORACLE_TIME_STRING_FORMATTER.format(instant)}'"
}
```

It will save the original behavior because of removed part from OracleDialect:

```
override fun processForDefaultValue(e: Expression<*>): String = when {
        e is LiteralOp<*> && (e.columnType as? IDateColumnType)?.hasTimePart == false -> "DATE ${super.processForDefaultValue(e)}"
        e is LiteralOp<*> && e.columnType is IDateColumnType -> "TIMESTAMP ${super.processForDefaultValue(e)}"
        else -> super.processForDefaultValue(e)
    }
```